app/src/androidTest/java/org/simple/clinic/medicalhistory/MedicalHistoryRepositoryAndroidTest.kt

```diff
@@ -102,17 +102,22 @@ class MedicalHistoryRepositoryAndroidTest {
     val newHistory = oldHistory.copy(hasHadHeartAttack = Yes)
     repository.save(newHistory, now)
 
-    val updatedHistory = repository.historyForPatientOrDefault(patientUuid).blockingFirst()
+    val updatedHistory = dao.historyForPatientImmediate(patientUuid)!!
 
     assertThat(updatedHistory.hasHadHeartAttack).isEqualTo(Yes)
     assertThat(updatedHistory.syncStatus).isEqualTo(SyncStatus.PENDING)
     assertThat(updatedHistory.updatedAt).isEqualTo(now)
   }
 
   @Test
-  fun when_medical_history_isnt_present_for_a_patient_then_an_empty_value_should_be_returned() {
-    val emptyHistory = repository.historyForPatientOrDefault(UUID.randomUUID()).blockingFirst()
-
+  fun when_medical_history_is_not_present_for_a_patient_then_an_empty_value_should_be_returned() {
+    val emptyHistoryUuid = UUID.fromString("b025b4e9-3907-4168-abd4-ab117739756d")
+    val emptyHistory = repository.historyForPatientOrDefault(
+        defaultHistoryUuid = emptyHistoryUuid,
+        patientUuid = UUID.randomUUID()
```

Comment:
Can we use the `patientUuid` instead of random UUID, please?

Reply:
Done.